diff --git a/camera/CameraParameters.cpp b/camera/CameraParameters.cpp
index 2e7018e..474f983 100644
--- a/camera/CameraParameters.cpp
+++ b/camera/CameraParameters.cpp
@@ -330,6 +330,7 @@ const char CameraParameters::SCENE_MODE_SPORTS[] = "sports";
 const char CameraParameters::SCENE_MODE_PARTY[] = "party";
 const char CameraParameters::SCENE_MODE_CANDLELIGHT[] = "candlelight";
 #ifdef STE_SAMSUNG_HARDWARE
+const char CameraParameters::SCENE_MODE_AQUA[] = "aqua";
 const char CameraParameters::SCENE_MODE_BACKLIGHT[] = "backlight";
 const char CameraParameters::SCENE_MODE_DUSKDAWN[] = "duskdawn";
 const char CameraParameters::SCENE_MODE_FALLCOLOR[] = "fallcolor";
diff --git a/include/camera/CameraParameters.h b/include/camera/CameraParameters.h
index abe1e6a..863949b 100644
--- a/include/camera/CameraParameters.h
+++ b/include/camera/CameraParameters.h
@@ -830,6 +830,7 @@ public:
     static const char SCENE_MODE_PARTY[];
     static const char SCENE_MODE_CANDLELIGHT[];
 #ifdef STE_SAMSUNG_HARDWARE
+    static const char SCENE_MODE_AQUA[];
     static const char SCENE_MODE_BACKLIGHT[];
     static const char SCENE_MODE_DUSKDAWN[];
     static const char SCENE_MODE_FALLCOLOR[];
